docs/source/faq.rst

```diff
@@ -2,3 +2,62 @@
 
 Implementation details, conventions, and FAQ
 ============================================
+
+Sign of the mixer Hamiltonian
+    In the original paper on QAOA (`Ref 1 <#references>`__), Farhi `et al` use :math:`\sum_i \hat{X}_i` as
+    the mixer Hamiltonian, with the initial state being its maximum eigenstate :math:`\left|+ \cdots +\right>`. 
+    In OpenQAOA, we instead choose our mixer Hamiltonian to be :math:`-\sum_i \hat{X}_i`, so that the initial state 
+    :math:`\left|+ \cdots +\right>` is now its minimum energy eigenstate. Conceptually this makes the analogy to adiabatic
+    computing clear, since we seek to transform from the ground state of the mixer Hamiltonian to the ground state of the cost Hamiltonian. 
+
+Implementation of circuit rotation angles
+    In quantum mechanics, the basic time evolution operator is :math:`\exp(-iHt)` for a Hamiltonian `H` and total
+    evolution time `t`. Generically, in the QAOA mixer Hamiltonian, the operator :math:`-X` is to be applied for a total time 
+    :math:`\beta`, which is one of the parameters we seek to optimise. We therefore need to implement the time evolution 
+    :math:`\exp(i\beta X)`, which can be achieved using the RX(:math:`\theta`) operator if we set :math:`\theta = -2\beta`. 
+
+    Similarly, the cost Hamiltonian operator :math:`\exp(-i\gamma hZ)` can be implemented via an RZ(:math:`\theta`) rotation, setting
```

Comment:
Maybe change the syntax to

You can verify these details in the methods that relate to the creation of these angles in ``openqaoa.qaoa_parameters.standardparams.QAOAVariationalStandardParams``.

Reply:
Resolved in latest commit